data/template.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
I dont' like to put this template under "data". That directory contains files that are used to create desktop software packages.

It's so short that it's probably easier to include it as a string inside the Python source file.

Reply:
I was not sure of the data usage.
Ok to include.